azlet/azertbot.py

```diff
@@ -113,11 +121,11 @@ def rotate(self, threshold: int = 14):
             logging.info(f"Starting renewal ...")
             self.create_certificate(domain_name=domain_name)
```

Comment:
hier müssten die Tags vorher ausgelesen werden und in das `create_certificate` mit reingegeben werden